src/main/scala/codacy/codesniffer/docsgen/Generator.scala

```diff
@@ -0,0 +1,65 @@
+package codacy.codesniffer.docsgen
+
+import better.files.File
+import codacy.codesniffer.docsgen.parsers._
+import com.codacy.plugins.api.results.Tool
+import play.api.libs.json.{Json, Writes}
+
+import scala.collection.parallel.ForkJoinTaskSupport
+import scala.collection.parallel.immutable.ParSeq
+
+class Generator() {
+
+  private[this] val toolName = Tool.Name("PHP Code Sniffer")
```

Comment:
This should be the same that we have as shortName inside codacy-plugins.
This should be a string without spaces and usually simple as an ID.

Reply:
This was the name that was defined before